main.py

```diff
@@ -258,6 +258,14 @@ async def ping(ctx):
     Checks if bot is alive
     '''
     await ctx.send('Hello world')
+# custome help commands
+@bot.command()
+async def cmds(ctx): # Help command
```

Comment:
changed this to help, overwrites discord.py's default help command

Reply:
both ways work, whether it is `>help` or `>cmds`
choose whichever you prefer